### PR TITLE
Feature/fix policy

### PR DESCRIPTION
**Pull request by limebell**
Contains 2 changes which are:
- Remove authorized miner checking in `BlockPolicy`.
- Do not append block when mining.

### CODE REVIEW

File: Lib9c/Policy/BlockPolicySource.cs [hunk 1]
```diff
@@ -234,11 +234,7 @@ public partial class BlockPolicySource
                     authorizedMinersPolicy,
                     defaultAlgorithm: chain => DifficultyAdjustment<NCAction>.BaseAlgorithm(
                         chain, BlockInterval, DifficultyStability, minimumDifficulty));
-            Func<Address, long, bool> isAllowedToMine = (address, index) => IsAllowedToMineRaw(
-                address,
-                index,
-                authorizedMinersPolicy,
-                permissionedMinersPolicy);
+            Func<Address, long, bool> isAllowedToMine = (address, index) => true;
```
Comment (riemannulus) [inline Lib9c/Policy/BlockPolicySource.cs:237]: I guess we are unnecessary this lambda. right?

Comment (limebell) [reply Lib9c/Policy/BlockPolicySource.cs:237]: Yes. This is an another temporary fix to run PBFT nodes ASAP///

Comment (riemannulus) [reply Lib9c/Policy/BlockPolicySource.cs:237]: Why can't we clear the lambda?

Comment (limebell) [reply Lib9c/Policy/BlockPolicySource.cs:237]: `BlockPolicy()` constructor takes `isAllowedToMine` parameter and it's default value is `null` not `true`.
https://github.com/planetarium/lib9c/blob/pbft/Lib9c/Policy/BlockPolicySource.cs#L260
https://github.com/planetarium/lib9c/blob/pbft/Lib9c/Policy/BlockPolicy.cs#L34

Comment (riemannulus) [reply Lib9c/Policy/BlockPolicySource.cs:237]: LGTM

**Pull request merged**